src/main/java/com/sarality/form/FormData.java

```diff
@@ -172,6 +170,13 @@ private String getValue(String fieldName) {
     return fieldValueMap.get(fieldName);
   }
 
+  private String convertEmptyToNull(String value) {
+    if (value.equals("")) {
```

Comment:
in that case, why dont we just replace the null check in the functions with an isEmpty check? Do we need to even convert empty to null?